package.json

```diff
@@ -3,11 +3,11 @@
     "version": "0.1.0",
     "description": "buckleScript bindings for RxJs v7",
     "scripts": {
```

Comment:
Do you think it is better to bind to global modules? This way we are making presumptions that should exist, if we are binding to local, guys would be able to run those just by installing repo dependencies

Reply:
You mean for `bs-platform`? I have rarely if ever seen using bsb locally. I used to see it in the early days. I would say use global bsb based on convention.

Reply:
unaware of such convention, alright then